backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/RequestVariablesSerializer.java

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.nimbusds.jose.util.StandardCharset;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.servlet.http.Part;
+
+public class RequestVariablesSerializer extends JsonSerializer<Map<String, Object>> {
+
+  private Object parseValue(Object value) {
+    if (value instanceof Part) {
+      Part part = (Part) value;
+      try {
+        return new BufferedReader(
+                new InputStreamReader(part.getInputStream(), StandardCharset.UTF_8))
+            .lines()
+            .collect(Collectors.joining("\n"));
+      } catch (IOException e) {
+        return "Unable to read file";
```

Comment:
Log this and then throw a different (runtime) exception. If we can't read the file, then audit logging has failed, and we need to bail on the whole request.